cli/setup-npmrc/register.ts

```diff
@@ -0,0 +1,120 @@
+import type { Command } from "commander"
+import kleur from "kleur"
+import { getSessionToken } from "lib/cli-config"
+import * as fs from "node:fs"
+import * as path from "node:path"
+import * as os from "node:os"
+
+const REGISTRY_URL = "npm.tscircuit.com"
+
+function findUserNpmrc(): string | null {
+  // Common locations for user-level .npmrc
+  const possiblePaths = [
+    // Unix/Linux/macOS: ~/.npmrc
+    path.join(os.homedir(), ".npmrc"),
+    // Windows: %USERPROFILE%\.npmrc
+    process.env.USERPROFILE
+      ? path.join(process.env.USERPROFILE, ".npmrc")
+      : null,
+    // npm config prefix location (for some setups)
+    process.env.NPM_CONFIG_USERCONFIG,
+  ].filter(Boolean) as string[]
+
+  // Return the first existing path, or the default home directory path
+  for (const npmrcPath of possiblePaths) {
+    if (fs.existsSync(npmrcPath)) {
+      return npmrcPath
+    }
+  }
+
+  // Return the default location even if it doesn't exist yet
+  return path.join(os.homedir(), ".npmrc")
+}
+
+function printManualInstructions(sessionToken: string) {
+  console.log(kleur.yellow("\nManual setup instructions:"))
+  console.log(kleur.gray("─".repeat(50)))
+  console.log("\n1. Open or create your user-level .npmrc file:")
+  console.log(kleur.cyan(`   ${path.join(os.homedir(), ".npmrc")}`))
+  console.log("\n2. Add the following line:")
+  console.log(kleur.cyan(`   //${REGISTRY_URL}/:_authToken=${sessionToken}`))
+  console.log(kleur.gray("\n─".repeat(50)))
+}
+
+export function registerSetupNpmrc(program: Command) {
+  program
+    .command("setup-npmrc")
+    .description(
+      "Configure user-level .npmrc with authentication for tscircuit private packages",
```

Comment:
user-level doesn't make sense

```suggestion
      "Configure your global .npmrc with authentication for tscircuit private packages",
```